infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -205,6 +205,9 @@
         blkid | awk '/ceph data/ { sub ("1:", "", $1); print $1 }'
       register: devices
       changed_when: false
+      when:
+        - devices is not defined
+        - devices | length > 0
```

Comment:
`devices is not defined` but his length would be `> 0` ?

Reply:
I need to wake up I guess